package.json

```diff
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/nearprotocol/near-explorer#readme",
   "dependencies": {
     "bootstrap": "^4.3.1",
+    "moment": "^2.24.0",
```

Comment:
FYI, Moment.js [used to be quite fat](https://github.com/moment/moment/issues/3376) the last time I checked it. [Luxon](https://moment.github.io/luxon/docs/manual/why.html) may be a better alternative, but I haven't checked it myself.

I think we can live with Moment.js for now, but we should keep this point in mind once.

Reply:
I'm pretty used to `Moment.js`. So used it. But yes, will take a look into that later.